addons/website_links/static/tests/tours/website_links.js

```diff
@@ -3,6 +3,33 @@
 import { registry } from "@web/core/registry";
 import { browser } from "@web/core/browser/browser";
 
+function fillSelect2(inputID, search) {
+    return [
+        {
+            content: "Click select2 form item",
+            trigger: `.o_website_links_utm_forms div.select2-container#s2id_${inputID} > .select2-choice`,
```

Comment:
Adapted `:has(+ ${inputID}` to another way of targetting the `div` because in 17.2, `:has` in tour step's triggers is solved by `hoot-dom/queryAll`, which executes the `:has`'s content in a distinct `querySelectorAll`.
Unfortunately, `querySelector("+ something")` is not valid, and `querySelector(":scope + something")` does not match either.